kolibri/core/assets/src/vue/content-renderer/download-button.vue

```diff
@@ -58,20 +58,16 @@
       downloadMediaText() {
         return this.$tr('downloadMedia');
       },
+      downloadLink() {
+        const filePath = this.availableFiles[0].storage_url;
```

Comment:
As in a button that downloads the main piece of content, and another that downloads the supplementary pieces of content?